packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -550,6 +550,11 @@ const schema: SchemaType<DbPost> = {
     }
   },
 
+  // (I'm not totally sure but this is my understanding of what this field is for):
+  // Back when we had a form where you could create a related question from a question post,
+  // you could set this to true to prevent the related question from appearing on the frontpage.
+  // Now that we've removed the form to create a related question, I think we can drop
+  // this field entirely?
   hiddenRelatedQuestion: {
     type: Boolean,
```

Comment:
I wanted to describe what this field is for now while I still have some context, because it will be much harder to figure that out in the future. But actually I think we can drop the field? If you agree, let me know and I will make an asana ticket so we don't forget.

Reply:
This matches my understanding of what it was used for, but I am not very confident. I agree we can remove it therefore, but am still not that confident. I would ask LW.

Reply:
Yes, I agree we can drop this field.